basic/src/test/java/org/jboss/hal/testsuite/test/configuration/general/TakeTourTestCase.java

```diff
@@ -0,0 +1,57 @@
+package org.jboss.hal.testsuite.test.configuration.general;
+
+import org.jboss.arquillian.drone.api.annotation.Drone;
+import org.jboss.arquillian.graphene.findby.ByJQuery;
+import org.jboss.arquillian.graphene.page.Page;
+import org.jboss.arquillian.junit.Arquillian;
+import org.jboss.hal.testsuite.category.Standalone;
+import org.jboss.hal.testsuite.fragment.WindowFragment;
+import org.jboss.hal.testsuite.page.home.HomePage;
+import org.jboss.hal.testsuite.util.Console;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.experimental.categories.Category;
+import org.junit.runner.RunWith;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+
+/**
+ * @author <a href="padamec@redhat.com">Petr Adamec</a>
+ */
+@RunWith(Arquillian.class)
+@Category(Standalone.class)
+public class TakeTourTestCase {
+
+    @Drone
+    public WebDriver browser;
+
+    @Page
+    public HomePage page;
+
+    @Before
+    public void setupWindow() {
+        browser.manage().window().maximize();
+        browser.navigate().refresh();
+        Console.withBrowser(browser).refreshAndNavigate(HomePage.class);
```

Comment:
`Console.withBrowser(browser).refreshAndNavigate(HomePage.class);` is IMHO the only needed line in this method or am I missing anything?